### ISSUE TITLE
Parameter sweep?

### ISSUE THREAD

**Issue by dbolser** [tensorflow/playground#91]
I know there are a lot of parameters to play with, but not too many to try a rough 'sweep' of all settings... I'm curious to try to categorise the results into working and non-working (converging and non-converging) to try to get some intuition about what works well in which situations.

Have you done this?

**Comment by emmanuelrouxfr** [comment]
Dear @dbolser, by reading your issue it seems that you may appreciate the fork we've been working on !

https://coolearning.github.io/playground/

The Coolearning interface allows interactive sweeping of the parameters by using MIDI user interface during training, and much more ! Like individual learning rate, activation functions and regularization for each neuron...

See you there :)
